test/functional/feature_blockfilterindex_prune.py

```diff
@@ -24,11 +24,7 @@ def run_test(self):
         self.log.info("check if we can access a blockfilter when pruning is enabled but no blocks are actually pruned")
         self.sync_index(height=200)
         assert_greater_than(len(self.nodes[0].getblockfilter(self.nodes[0].getbestblockhash())['filter']), 0)
-        # Mine two batches of blocks to avoid hitting NODE_NETWORK_LIMITED_MIN_BLOCKS disconnection
-        self.generate(self.nodes[0], 250)
-        self.sync_all()
-        self.generate(self.nodes[0], 250)
-        self.sync_all()
+        self.generate(self.nodes[0], 500)
```

Comment:
Why is this change correct?

Reply:
Because preventing disconnections due to NODE_NETWORK_LIMITED_MIN_BLOCKS makes little sense  when there is just one node involved in the test.